src/files-action.js

```diff
@@ -37,8 +36,8 @@ window.addEventListener('DOMContentLoaded', () => {
 				window.location = generateUrl(`/apps/contacts/import?file=${absPath}`)
 			},
 		})
+	// to be changed
 		OCA.Files.fileActions.setDefault(mime, name)
```

Comment:
@skjnldsv  can you please help me with this part and how to adapt i correctly. The doc says: _" ``OCA.Files``: Everything removed but Sidebar and Settings. See the Added API section for replacements."_

But Im not sure how to do it.

Reply:
Read the docs on the requirements and available properties: https://nextcloud-libraries.github.io/nextcloud-files/classes/FileAction.html

Reply:
@GretaD https://nextcloud-libraries.github.io/nextcloud-files/classes/FileAction.html#default. Pass it as `default: 'default',` in the file action object above.